--- /usr/share/pcl-1.8/PCLConfig.cmake	2017-02-28 23:33:04.000000000 +0100
+++ ./PCLConfig.cmake	2017-07-19 14:59:45.988250023 +0200
@@ -743,7 +743,7 @@
   get_filename_component(PCL_ROOT "${PCL_DIR}" PATH)
 else(WIN32 AND NOT MINGW)
 # PCLConfig.cmake is installed to PCL_ROOT/share/pcl-x.y
-  set(PCL_ROOT "/usr/local")
+  set(PCL_ROOT "/usr")
 endif(WIN32 AND NOT MINGW)
 
 # check whether PCLConfig.cmake is found into a PCL installation or in a build tree
